### ISSUE TITLE
gnutls coverage build failure with profile format version mismatch

### ISSUE THREAD

**Issue by ueno** [google/oss-fuzz#6473]
I am seeing this error in the recent couple of coverage build failures:
```console
Step #5: warning: /workspace/out/libfuzzer-coverage-x86_64/dumps/gnutls_reverse_idna_parser_fuzzer.2988909203736624255_0.profraw: unsupported instrumentation profile format version
Step #5: error: no profile can be merged
Step #5: warning: /workspace/out/libfuzzer-coverage-x86_64/dumps/gnutls_idna_parser_fuzzer.14371750675475251198_0.profraw: unsupported instrumentation profile format version
Step #5: error: no profile can be merged
```
It looks similar to https://github.com/google/fuzzbench/issues/590, but I have no clue whether this can be fixed in the project alone. Any advice would be appreciated.

**Comment by jonathanmetzman** [comment]
I assume it is related to https://github.com/google/oss-fuzz/blob/master/projects/gnutls/Dockerfile#L19
I would guess this problem affects all projects still on xenial.
To solve this, I think gnutls needs to use focal. I think the reason it didn't just work on focal was because it couldn't find libnettle. I tried fixing this myself, but stopped because gnutls took a really long time to build and get to that failure so the feedback loop was too long and I had other projects I needed to fix.
